docs/reference/index-modules/slowlog.asciidoc

```diff
@@ -21,6 +21,8 @@ The following is an example of a search event in the slow log:
 
 TIP: If a call was initiated with an `X-Opaque-ID` header, then the ID is automatically included in Search slow logs in the **elasticsearch.slowlog.id** field. See <<x-opaque-id,X-Opaque-Id HTTP header>> for details and best practices.
 
+TIP: You can use the `index.search.slowlog.include.user` or `index.indexing.slowlog.include.user` settings to append `user.*` and `auth.type` fields to slow log entries. These fields contain information about the user who triggered the request.
```

Comment:
we should use tips sparingly - I'd consider this to be something that could be included in the "core" docs. is there a reason why you moved it?

Reply:
🙂 Because it applies to both search+indexing & you had moved it only under search while it references both so I was bringing it back up to the shared notes region. Agreed on not doing a tip, I was following your x-opaque-id format.